antenna/throttler.py

```diff
@@ -113,11 +113,11 @@ def throttle(self, raw_crash):
                 if rule.percentage == 100 or (random.random() * 100.0) <= rule.percentage:  # nosec
                     response = ACCEPT
                 else:
-                    response = DEFER
+                    response = REJECT
                 return response, rule.rule_name, rule.percentage
 
         # None of the rules matched, so we defer
-        return DEFER, 'NO_MATCH', 0
+        return REJECT, 'NO_MATCH', 0
```

Comment:
This is also a change--if the crash report matched none of the rules, then we reject it.

However, it has no effect on anything since we added a catch-all "accept everything" rule to the end.

Why do it that way? I wanted to make it easier to think about and for other groups to use Antenna. I want to get rid of the notion of DEFER altogether. Seems like if a crash report matches none of the rules set by some non-Mozilla group, it makes more sense to reject it.